### ISSUE TITLE
IConnectionParameters.connect_timeout has no effect

### ISSUE THREAD

**Issue by tmtron** [vitaly-t/pg-promise#682]
### Expected behavior
it should be possible to set the connection-timeout via [IConnectionParameters](https://github.com/vitaly-t/pg-promise/blob/master/typescript/pg-subset.d.ts#L79)

### Actual behavior
the `connect_timeout` has no effect

### Steps to reproduce
I think the parameter name is wrong.
node-postgres will look for `connectionTimeoutMillis` in the config object and then assign it to the internal `connect_timeout` 
from [connection-parameters.js#L73](https://github.com/brianc/node-postgres/blob/e4578d2c7bca60cfb02ec004daa22c8ff810140e/lib/connection-parameters.js#L73):
```
var ConnectionParameters = function (config) {
  ...
  this.connect_timeout = Math.floor(config.connectionTimeoutMillis / 1000)
}
```
So I the fix should be to rename [`connect_timeout`](https://github.com/vitaly-t/pg-promise/blob/master/typescript/pg-subset.d.ts#L99) in `IConnectionParameters` to `connectionTimeoutMillis`

workaround for now:

```
    const config: IConnectionParameters = {
      user: 'myuser',
      port: 5444
    };
    (config as any).connectionTimeoutMillis = 1000;

```

### Environment

* Version of pg-promise: 10.3.1
* OS type (Linux/Windows/Mac): Win 10 64-bit
* Version of Node.js: 12.12.0

**Comment by vitaly-t** [comment]
So #681 might be related, even though the example there was simply invalid.

**Comment by vitaly-t** [comment]
I can't figure this mess out myself :smile:

Opened [issue against the driver](https://github.com/brianc/node-pg-pool/issues/140).

**Comment by vitaly-t** [comment]
So, to avoid relying on the driver's feedback, I had a chance to do some testing here, and it seems that `connectionTimeoutMillis` is indeed the only right way to do it currently.

I will be patching the library shortly.

**Comment by vitaly-t** [comment]
Fixed and released in [v10.3.4](https://github.com/vitaly-t/pg-promise/releases/tag/10.3.4).
